sdk/textanalytics/out/azure/ai/textanalytics/_generated/aio/operations_async/_text_analytics_client_operations_async.pyi

```diff
@@ -0,0 +1,13 @@
+# Stubs for azure.ai.textanalytics._generated.aio.operations_async._text_analytics_client_operations_async (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any, Optional
+
+class TextAnalyticsClientOperationsMixin:
+    async def entities_recognition_general(self, documents: Any, model_version: Optional[Any] = ..., show_stats: Optional[Any] = ..., *, cls: Optional[Any] = ..., **kwargs: Any): ...
```

Comment:
What exactly are valid values for `show_stats` ?

Reply:
`True` and `False` unless I'm mistaken :).